### PR TITLE
NotesTab

### PR DESCRIPTION
**Pull request by eikonomega**
Fulfills #572

### CODE REVIEW

**Comment by dmfalke** [comment]
Doing some functional testing. Looks great! One thing I noticed is the character count in the description collides with the text.
image

File: package.json [hunk 1]
```diff
@@ -2,29 +2,31 @@
...
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@mui/styles": "^5.0.1",
     "@types/debounce-promise": "^3.1.3",
-    "@veupathdb/components": "^0.8.8",
     "@veupathdb/core-components": "^0.2.43",
+    "@veupathdb/web-common": "^0.1.10",
```
Comment (dmfalke) [inline package.json:13]: This should not be a dependency. It is only needed as a devDependency, and will create a circular dependency if left here.
 
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -2,29 +2,31 @@
...
     "fp-ts": "^2.9.3",
     "io-ts": "^2.2.13",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
     "monocle-ts": "^2.3.11",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-draggable": "^4.4.3",
     "react-resizable": "^3.0.4",
+    "react-router": "^5.2.1",
```
Comment (dmfalke) [inline package.json:28]: I think only `react-router-dom` is needed.

```suggestion
```

File: public/index.html [hunk 3]
```diff
@@ -9,6 +9,12 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
```
Comment (dmfalke) [inline public/index.html:17]: Where are these fonts being used?

**Review by dmfalke — CHANGES_REQUESTED** [review]
Once the dependencies are cleaned up, and the glitch I mentioned above is addressed, I think this will be good to go!

File: package.json (hunk 2 above)
Comment (eikonomega) [reply package.json:28]: I see a number of places using imports from react-router.

For ex. see `MapVeuContainer.tsx` or `AnalysisPanel.tsx`

**Review by dmfalke — APPROVED** [review]

**Pull request merged**